### ISSUE TITLE
FCOS train loss NAN

### ISSUE THREAD

**Issue by huangxiaosa** [Stick-To/Object-Detection-API-Tensorflow#49]
hi, I train FCOS and Centernet use same dataset, Centernet sames ok while FCOS loss is NAN at the early beginning. I tried to use gradient clip,but the loss would keep high.

**Comment by Nathan-For** [comment]
> hi, I train FCOS and Centernet use same dataset, Centernet sames ok while FCOS loss is NAN at the early beginning. I tried to use gradient clip,but the loss would keep high.

can you chare how to train?

**Comment by phamkhactu** [comment]
Hi @letcoN , I am a newbie. I have the same problem with the Centernet. Loss always nan. Can you share pre-trained weight and share how to train. I think maybe my step preparation fault. Thanks in advance!!
